pulsar-client-cpp/README.md

```diff
@@ -277,3 +277,23 @@ ${PULSAR_PATH}/pulsar-test-service-stop.sh
 It's recommended to install [LLVM](https://llvm.org/builds/) for `clang-tidy` and `clang-format`. Pulsar C++ client use `clang-format` 5.0 to format files, which is a little different with latest `clang-format`.
 
 We welcome contributions from the open source community, kindly make sure your changes are backward compatible with GCC 4.8 and Boost 1.53.
+
+### Install `clang-format` on macOS
+
+Since the homebrew-core doesn't have `clang-format@5`, here is a tap to install `clang-format@5` in your macOS.
```

Comment:
```suggestion
`homebrew-core` does not have `clang-format@5`. You can install `clang-format@5` on your macOS using the tap below.
```

1. Do you mean this?


2. Do not use contractions in technical information. Reason: https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.qa5rwiehb245

Reply:
Thanks for your suggestion. Fixed.